lib/standalone.js

```diff
@@ -1,25 +1,24 @@
 'use strict';
 
-const createStylelint = require('./createStylelint');
-const createStylelintResult = require('./createStylelintResult');
 const debug = require('debug')('stylelint:standalone');
 const fastGlob = require('fast-glob');
+const fs = require('fs');
+const globby = require('globby');
+const normalizePath = require('normalize-path');
+const path = require('path');
+
+const createStylelint = require('./createStylelint');
```

Comment:
[note] Separated by a blank line to distinguish external and internal importing.